chapter-2/README.md

```diff
@@ -54,7 +54,9 @@ Run the script by copying this command into your terminal from inside the `chapt
 
 By running this script, you will fetch all the required images and then load them into every node of your KinD cluster. If you are running the examples on a Cloud Provider, this might not be worth it as Cloud Providers with Gigabyte connections to container registries might fetch these images in a matter of seconds.
 
+**Note:** If you are running Docker Desktop on MacOS, please make sure that the virtual disk has at least 15GB of available space. You can modify the value of the Virtual Disk limit in the Settings -> Resources menu.
```

Comment:
@zouyingjie I am not sure that we need 15GB of space. This might be because you were already using part of the disk space that you had configured before. I would change this text to say, if you find issue "X" I don't know what error did you got exactly, then increase the Virtual Disk Limit in Settings -> Resources "  Does that make sense?

Reply:
After I ran the app, the virtual disk free space decreased by 13Gb, which I estimated based on this number. But you're right, just add a note to the issue.

Reply:
yeah.. but running that script download container images for around 1.5GB.. so 15GB sounds like too much